# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nfs-utils/hotfix-const.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nfs-utils-2.8.2/support/nsm/file.c.vanilla	2024-12-14 15:58:59.419938034 +0100
+++ nfs-utils-2.8.2/support/nsm/file.c	2024-12-14 15:59:17.343271279 +0100
@@ -184,7 +184,7 @@
 nsm_make_temp_pathname(const char *pathname)
 {
 	size_t size;
-	char *path, *base;
+	char *path; const char *base;
 	int len;
 
 	size = strlen(pathname) + sizeof(".new") + 1;
